modules/yieldlabBidAdapter.js

```diff
@@ -142,6 +142,13 @@ export const spec = {
           }
         }
 
+        if (isNative(bidRequest, adType)) {
+          const url = `${ENDPOINT}/d/${matchedBid.id}/${bidRequest.params.supplyId}/?ts=${timestamp}${extId}${gdprApplies}${gdprConsent}${pvId}`
```

Comment:
We want to keep this address for a short time, since we have publisher who use the workaround solution. 
Later this url will be replaced by `/md/{adslotId}/{userId}?ids={externalUserIds}&mdata={true|false}&pvid={true|false}.` or removed by at all because this link is part of `imptrackers`
```
"imptrackers": [
                "http://localhost:8082/ve?d=ODE9ZSYzMTIxMjM4OTI3MjYxPXN0JmY3MGNiZWYyMWZhYi05MWNiLWZkZDQtMzQ0Zi01YTMwMjcxZT1kaXQmMj12",
                "http://localhost:8080/md/12548955/9efa4e76-2030-4f04-bb9f-322541f8d611?mdata=false&pvid=false&ids=x:1,y:2",
                "http://localhost:8081/imp?s=13216&d=2171514&a=12548955&ts=1627298321213&tid=e17203a5-f443-4ddf-bc19-baf12febc07f&l=0"
            ]
```